macantine/etl/utils.py

```diff
@@ -205,6 +204,9 @@ def map_sectors():
     """
     Populate the details of a sector, given its id
     """
+
+    from api.serializers import SectorSerializer  # avoir circular import
```

Comment:
```suggestion
    from api.serializers import SectorSerializer  # avoid circular import
```

Reply:
the `map_sectors` method could be moved elsewhere (to avoid the import error).
for instance in the `sector/serializers.py` file

Reply:
haha je me rends compte que j'ai écrit en anglais :sweat_smile: (l'influence OFF :eyes)